pkg/kubelet/dockertools/labels_test.go

```diff
@@ -40,10 +40,11 @@ func TestLabels(t *testing.T) {
 				Command: []string{"action1", "action2"},
 			},
 			HTTPGet: &api.HTTPGetAction{
-				Path:   "path",
-				Host:   "host",
-				Port:   intstr.FromInt(8080),
-				Scheme: "scheme",
+				Path:        "path",
+				Host:        "host",
+				Port:        intstr.FromInt(8080),
+				Scheme:      "scheme",
+				HTTPHeaders: nil,
```

Comment:
is this needed?  nil should be the default, and it's not relevant to the test...